### PR TITLE
fix(stubs/click): add the `no_args_is_help` argument annotaion for click

### LINKED ISSUE #5113: Unexpected keyword argument "no_args_is_help" for "command" using click
I am using [click](https://github.com/pallets/click) to parse command-line arguments. But when running `mypy test.py`, it complains an unexpected keyword argument `no_args_is_help`.

Codes below are in the "test.py" file:

```python
#!/usr/bin/env python3
# _*_ coding: utf-8 _*_

import click


@click.command("hello", no_args_is_help=True)
def hello():
    click.echo("Hello")


if __name__ == "__main__":
    hello(prog_name="hello")

```

And I found that the stubs file of `click.decorators.command` wasn't contains `no_args_is_help`.

https://github.com/python/typeshed/blob/30fc2b8cd0420a662615e2a2b6dd28d091618239/stubs/click/click/decorators.pyi#L21-L33

This is the [documentation](https://click.palletsprojects.com/en/7.x/api/#click.Command) of `no_args_is_help`.

### PR DESCRIPTION
**Pull request by Pagliacii**
close #5113

### CODE REVIEW

**Review by Akuli — COMMENTED** [review]
Should `def group(` in `decorators.pyi`, starting at line 38, have this too?

**Comment by Pagliacii** [comment]
https://github.com/python/typeshed/blob/97c6806582bed88afae2bb4183d10fe45fbad6fc/stubs/click/click/decorators.pyi#L37-L58

The `group` decorator has a `no_args_is_help` already.

**Comment by Akuli** [comment]
Seems like `no_args_is_help` should be deleted from `MultiCommand`, as it now inherits it from `Command`, and in `group()` it should be moved under `# Command`. Seems like I can't add suggestions to those parts of the code by doing a code review.

**Comment by Pagliacii** [comment]
Ok, I see. I updated the code to remove this argument from the `MultiCommand` definition and rearrange the order of `group` arguments.

File: stubs/click/click/core.pyi [hunk 1]
```diff
@@ -151,7 +153,6 @@ _T = TypeVar("_T")
 _F = TypeVar("_F", bound=Callable[..., Any])
 
 class MultiCommand(Command):
-    no_args_is_help: bool
```
Comment (Akuli) [inline stubs/click/click/core.pyi:154]: I guess this should also be deleted from `MultiCommand`'s `__init__`?

Comment (Akuli) [reply stubs/click/click/core.pyi:154]: Actually, we shouldn't delete this from `MultiCommand`, it should be in both classes like [it is here](https://github.com/pallets/click/blob/master/src/click/core.py)

Comment (Pagliacii) [reply stubs/click/click/core.pyi:154]: Oh, I didn't see this...

Wait a minute.

Comment (Pagliacii) [reply stubs/click/click/core.pyi:154]: > Actually, we shouldn't delete this from `MultiCommand`, it should be in both classes like [it is here](https://github.com/pallets/click/blob/master/src/click/core.py?rgh-link-date=2021-04-10T14%3A08%3A20Z)

Yes, you are right, we should keep this argument in MultiCommand constructor.

**Comment by Pagliacii** [comment]
And I think we should keep the `no_args_is_help` stay after `# MultiCommand` because the `Group` class inherits from `MultiCommand`.

https://github.com/pallets/click/blob/3f7319081113a0bdfa8fa0b8c4ff018a8f5e5278/src/click/core.py#L1624

**Review by Akuli — APPROVED** [review]

**Pull request merged**